ui/search/nuxeo-search-form-layout.js

```diff
@@ -95,7 +96,12 @@ import { RoutingBehavior } from '../nuxeo-routing-behavior.js';
         /**
          * The base url for the layout href.
          */
-        hrefBase: String,
+        hrefBase: {
+          type: String,
+          value() {
+            return config.get('layouts.search.hrefBase');
```

Comment:
would it make sense to introduce already a `config.get('layouts.search.hrefBase') || (config.get('layouts.hrefBase') && (config.get('layouts.hrefBase') + '/search')` so we can reuse the same config for the different layouts later?

Reply:
I think it would be best to do this at a later stage, once we actually have a need for a `layouts.hrefBase`, and outside of the scope of bug fixing.

Reply:
But the change makes perfect sense.

Reply:
@nelsonsilva, created the following tickets to implemented this:
- https://jira.nuxeo.com/browse/ELEMENTS-1444
- https://jira.nuxeo.com/browse/ELEMENTS-1445